### DISCUSSION TITLE
Browser did something unexpected

### DISCUSSION THREAD

**Discussion by ghost** [community/community#35786] [Other Feature Feedback, Questions, & Ideas]
Hello everyone

I am trying to connect Azure Board with GitHub but I get next error:

> Your browser did something unexpected. Please try again. If the error continues, try disabling all browser extensions.

I tested in different browsers but same error. I checked in Settings/Applications but I have one.
Any idea?

thanks

**Comment by jfullstackdev** [comment]
how about Incognito ? 

then check if you have that still in Incognito

**Reply by ghost** [reply]
Same error! I tested Chrome and Edge both incognito too.

**Comment by japneetsingh035** [comment]
Hi @quimavila, I am having same issue! On every browser

**Reply by ghost** [reply]
Hi @japneetsingh035,

could you solve it?

**Comment by JimNewaz** [comment]
It might be a reason for any extensions. If you are using any VPN or Adblock extensions, turn off them and try again. Also, enable private internet access. Hope it will work.

**Reply by ghost** [reply]
I don't use any VPN and Adblock. It seems a bug because appears a new error now:

`Failed to set events: commit_comment, content_reference, issues, issue_comment, pull_request, push, repository.`

**Comment by ghost** [comment]
Hello

it seems Auzre Boards is connected to my account right:
`You have already purchased this app on GitHub Marketplace.`

When I try to connect to any repo, then appears next error:
`Your browser did something unexpected. Please try again. If the error continues, try disabling all browser extensions.`

I try troubleshooting steps (clear cache, different browsers, incognito ...). In fact, I try in another computer, same result.

Any idea?

**Comment by svenrienstra** [comment]
I have the same issue, I've also openend a question on Azure side: https://developercommunity.visualstudio.com/t/Cant-install-Azure-Boards-Github-app/10174722?space=21&q=github&sort=newest&entry=myfeedback

**Reply by ghost** [reply]
I opened a support ticket on Github too. It seems a bug and they are working on it.

**Reply by github4453** [reply]
I am very interested as I can not get this to work for me

**Reply by ghost** [reply]
@github4453 , it seems bug was solved. I tryed to connect app and it works fine.

**Comment by ghost** [comment]
Hello everyone

it seems bug was solved. I tryed to connect app and it works fine.

Regards

**Comment by pwnspirit** [comment]
Considering old issues still can _solve_ such problems, It did. Consider requesting in **private mode**, **disable extensions** or **change to private internet**.
